src/app/containers/ATIAnalytics/params/cpsAssetPage/buildParams.js

```diff
@@ -0,0 +1,43 @@
+import path from 'ramda/src/path';
+import { buildATIPageTrackPath } from '../../atiUrl';
+import { getPublishedDatetime } from '../../../../lib/analyticsUtils';
+
+export const buildCPSATIParams = (pageData, requestContext, serviceContext) => {
+  const { platform, statsDestination } = requestContext;
+  const {
+    atiAnalyticsAppName,
+    atiAnalyticsProducerId,
+    service,
+  } = serviceContext;
+
+  const { metadata, promo } = pageData;
+
+  // ATI gets it's "Chapter 1" value from a prefix to the page identifier
+  // eg embedded_media::pidgin.embedded_media.media_asset.49529724.page
```

Comment:
Is this comment necessary? Could the code not be more self documenting with something like:

```
const getChapter1 = page => page.split('.')[1];
...

const chapter1 = isValidPage ? getChapter1(page) : false; (I think it's false if not valid)
```